lib/playground.dart

```diff
@@ -137,6 +139,11 @@ class Playground implements GistContainer, GistController {
     bind(editableGist.property('description'), titleEditable.textProperty);
     editableGist.onDirtyChanged.listen((val) {
       titleEditable.element.classes.toggle('dirty', val);
+      if (val) {
+        share.setInnerHtml("Share...");
+      } else {
+        share.setInnerHtml("Embed...");
```

Comment:
Would a slightly longer name help here? Is the user going to know what `Embed...` does?

Let's use the `…` char instead of three dots, here and for the rest of the PR.